plutus-core/plutus-core/test/Pretty/Golden/Readable/Examples/List/omapList.golden

```diff
@@ -1 +1,55 @@
-/\a -> \(f : a -> a) -> (/\a -> /\b -> \(f : (a -> b) -> a -> b) -> (/\a -> \(s : (\a -> ifix (\(self :: * -> *) -> \a -> self a -> a) a) a) -> unwrap s s) {a -> b} (iwrap (\(self :: * -> *) -> \a -> self a -> a) (a -> b) (\(s : (\a -> ifix (\(self :: * -> *) -> \a -> self a -> a) a) (a -> b)) -> \(x : a) -> f ((/\a -> \(s : (\a -> ifix (\(self :: * -> *) -> \a -> self a -> a) a) a) -> unwrap s s) {a -> b} s) x))) {list a} {list a} (\(rec : list a -> list a) -> \(xs : list a) -> (/\a -> \(x : list a) -> /\r -> \(z : r) -> \(f : a -> list a -> r) -> (Left chooseList) {a} {unit -> r} x (\(u : unit) -> z) (\(u : unit) -> f ((Left headList) {a} x) ((Left tailList) {a} x)) ()) {a} xs {list a} xs (\(x : a) -> \(xs' : list a) -> (Left mkCons) {a} (f x) (rec xs')))
\ No newline at end of file
+/\a ->
+  \(f : a -> a) ->
+    (/\a
+       b ->
+       \(f : (a -> b) -> a -> b) ->
+         (/\a ->
+            \(s : (\a -> ifix (\(self :: * -> *) a -> self a -> a) a) a) ->
+              unwrap s s)
+           {a -> b}
+           (iwrap (\(self :: * -> *)
+                    a ->
+                     self
+                       a -> a) (a -> b) (\(s
```

Comment:
The breaking here is weird: we break the first argument of `iwrap` but then the second goes on one line and then the third starts on one line and then breaks. And that looks bad because we _align_ with the third which is now far into the page.

I think we maybe just need to make every construct with more than one child break so that if it doesn't fit on one line then we `indent . align . vcat` the children.